service/gradle.lockfile

```diff
@@ -234,6 +256,7 @@ org.bouncycastle:bcpkix-jdk15on:1.69=productionRuntimeClasspath,runtimeClasspath
 org.bouncycastle:bcprov-ext-jdk15on:1.69=productionRuntimeClasspath,runtimeClasspath,testRuntimeClasspath
 org.bouncycastle:bcprov-jdk15on:1.69=productionRuntimeClasspath,runtimeClasspath,testRuntimeClasspath
 org.bouncycastle:bcutil-jdk15on:1.69=productionRuntimeClasspath,runtimeClasspath,testRuntimeClasspath
+org.broadinstitute.dsde.workbench:sam-client_2.12:0.1-61135c7=productionRuntimeClasspath,runtimeClasspath,testRuntimeClasspath
```

Comment:
Curious, why there is a dependency on an old Sam client?

Reply:
I think transitive dependency from landing zone library..